src/neo4j_genai/filters.py

```diff
@@ -55,11 +55,11 @@ def safe_field_cypher(field_name: str) -> str:
         escaped_field = field_name.replace("`", "``")
         return f"`{escaped_field}`"
 
-    def lhs(self, field):
+    def lhs(self, field: str) -> str:
         safe_field_cypher = self.safe_field_cypher(field)
         return f"{self.node_alias}.{safe_field_cypher}"
 
-    def cleaned_value(self, value):
+    def cleaned_value(self, value: str) -> str:
```

Comment:
Here `value` should be `Any`. We might want to compare stuff to float or anything, we can also have lists.

Reply:
If so should it be `list[float | int | str]` as you suggested below?